src/components/infoBox.tsx

```diff
@@ -14,21 +14,25 @@ interface InfoBoxProps extends React.HTMLAttributes<HTMLDivElement> {
   title?: string;
   textSize?: "small" | "medium" | "large";
   addClasses?: string;
+  size?: "small";
```

Comment:
In addition to checking for null, you could make this more obvious by adding null as an option: "small" | null